examples/llm/vdb_upload/vdb_config.yaml

```diff
@@ -0,0 +1,305 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+vdb_pipeline:
+  embeddings:
+    isolate_embeddings: false
+    model_kwargs:
+      force_convert_inputs: true
+      model_name: "all-MiniLM-L6-v2"
```

Comment:
Updating model_name in vdb_config.yml doesn't change the default `model_name` config in the pipeline. Reproduce by changing `model_name` to `intfloat/e5-large-v2`
Running the cli uses the pipeline model_name 
`python examples/llm/main.py vdb_upload pipeline   --vdb_config_path examples/llm/main/vdb_config.yaml `

Reply:
Good catch; fixing.